users/ninjonas/process_records.c

```diff
@@ -42,6 +47,13 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
       }
       break;
 
+    // Sends QMK make command to compile all keyboards
+    case M_MALL:
+     if (record->event.pressed) {
+        SEND_STRING("rm -f *.hex && rm -rf .build/ && make crkbd:ninjonas lily58:ninjonas hotdox:ninjonas pinky/3:ninjonas " SS_TAP(X_ENTER));
```

Comment:
```suggestion
        SEND_STRING("rm -f *.hex && rm -rf .build/ && make crkbd:ninjonas lily58:ninjonas hotdox:ninjonas pinky/3:ninjonas\n");
```

The newline character `\n` will send the Enter key :)